src/app/query/stacks/fees/fees.hooks.ts

```diff
@@ -61,7 +61,12 @@ export function useFeeEstimations(txByteLength: number | null, txPayload: string
     if (!isLoading && isError) {
       return { estimates: defaultFeeEstimations, calculation: FeeCalculationType.Default };
     }
-    if (!isLoading && (!!txFeeEstimation?.error || !feeEstimations?.length) && txByteLength) {
+    if (
+      txFeeEstimation &&
+      !isLoading &&
+      (!!txFeeEstimation?.error || !feeEstimations?.length) &&
+      txByteLength
+    ) {
```

Comment:
This is quite a difficult to read expression. Could it be given a readable function name?

Reply:
Sure, I can, I am trying to remember why this condition would happen? 🤔